pkgs/development/python-modules/myfitnesspal/default.nix

```diff
@@ -11,14 +11,18 @@ buildPythonPackage rec {
     sha256 = "c2275e91c794a3569a76c47c78cf2ff04d7f569a98558227e899ead7b30af0d6";
   };
 
-  # Remove overly restrictive version constraints on keyring and keyrings.alt
+  # Remove overly restrictive version constraints
   postPatch = ''
     sed -i 's/keyring>=.*/keyring/' requirements.txt
     sed -i 's/keyrings.alt>=.*/keyrings.alt/' requirements.txt
+    sed -i 's/rich>=.*/rich/' requirements.txt
   '';
 
-  checkInputs = [ mock nose ];
-  propagatedBuildInputs = [ blessed keyring keyrings-alt lxml measurement python-dateutil requests six ];
+  checkInputs = [ pytestCheckHook mock nose ];
```

Comment:
Another matter of taste: I personally try to order these inputs by the phases they occur in. Following that `checkInputs` should occur further down and be grouped together with `disabledTests`, since both concern the checkPhase.

Reply:
Sounds good, moved it down.